plugin/pxr/maya/lib/usdMaya/exportCommand.cpp

```diff
@@ -13,308 +13,21 @@
-                   MSyntax::kBoolean);
-    syntax.addFlag("-dms",
-                   UsdMayaJobExportArgsTokens->defaultMeshScheme.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-vis",
-                   UsdMayaJobExportArgsTokens->exportVisibility.GetText(),
-                   MSyntax::kBoolean);
-    syntax.addFlag("-ero" ,
-                   UsdMayaJobExportArgsTokens->exportReferenceObjects.GetText(),
-                   MSyntax::kBoolean);
-    syntax.addFlag("-skl",
-                   UsdMayaJobExportArgsTokens->exportSkels.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-skn",
-                   UsdMayaJobExportArgsTokens->exportSkin.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-psc",
-                   UsdMayaJobExportArgsTokens->parentScope.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-ro",
-                   UsdMayaJobExportArgsTokens->renderableOnly.GetText(),
-                   MSyntax::kNoArg);
-    syntax.addFlag("-dc",
-                   UsdMayaJobExportArgsTokens->defaultCameras.GetText(),
-                   MSyntax::kNoArg);
-    syntax.addFlag("-rlm",
-                   UsdMayaJobExportArgsTokens->renderLayerMode.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-k",
-                   UsdMayaJobExportArgsTokens->kind.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-com",
-                   UsdMayaJobExportArgsTokens->compatibility.GetText(),
-                   MSyntax::kString);
-
-    syntax.addFlag("-chr",
-                   UsdMayaJobExportArgsTokens->chaser.GetText(),
-                   MSyntax::kString);
-    syntax.makeFlagMultiUse(UsdMayaJobExportArgsTokens->chaser.GetText());
-
-    syntax.addFlag("-cha",
-                   UsdMayaJobExportArgsTokens->chaserArgs.GetText(),
-                   MSyntax::kString, MSyntax::kString, MSyntax::kString);
-    syntax.makeFlagMultiUse(UsdMayaJobExportArgsTokens->chaserArgs.GetText());
-
-    syntax.addFlag("-mfc",
-                   UsdMayaJobExportArgsTokens->melPerFrameCallback.GetText(),
-                   MSyntax::kNoArg);
-    syntax.addFlag("-mpc",
-                   UsdMayaJobExportArgsTokens->melPostCallback.GetText(),
-                   MSyntax::kNoArg);
-    syntax.addFlag("-pfc",
-                   UsdMayaJobExportArgsTokens->pythonPerFrameCallback.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-ppc",
-                   UsdMayaJobExportArgsTokens->pythonPostCallback.GetText(),
-                   MSyntax::kString);
-    syntax.addFlag("-v",
-                   UsdMayaJobExportArgsTokens->verbose.GetText(),
-                   MSyntax::kNoArg);
-
-    // These are additional flags under our control.
-    syntax.addFlag("-fr", "-frameRange", MSyntax::kDouble, MSyntax::kDouble);
-    syntax.addFlag("-ft", "-frameStride", MSyntax::kDouble);
```

Comment:
All of this has now been moved to the base class, but here you'll see the short flag conflict (lines 156 and 164)